test/language/expressions/async-arrow-function/eval-var-scope-syntax-err.js

```diff
@@ -1,11 +1,11 @@
 // This file was procedurally generated from the following sources:
 // - src/function-forms/eval-var-scope-syntax-err.case
-// - src/function-forms/error/async-arrow-function.template
+// - src/function-forms/error-no-strict/async-arrow-function.template
 /*---
-description: sloppy direct eval in params introduces var (async arrow function expression)
+description: sloppy direct eval in params introduces var (async arrow function expression in sloppy code)
 esid: sec-async-arrow-function-definitions
 features: [default-parameters, async-functions]
-flags: [generated, noStrict, async]
```

Comment:
The `noStrict` flag is incorrectly removed here due to flags misformatting in the template.